OIPA/iati/models.py

```diff
@@ -597,6 +599,41 @@ class DocumentLinkTitle(models.Model):
         content_type_field='related_content_type',
         object_id_field='related_object_id')
 
+class DocumentSearch(models.Model):
+    document = models.OneToOneField('Document')
+    content = VectorField()
+    text = VectorField()
+    last_reindexed = models.DateTimeField()
+
+class Document(models.Model):
+    document_link = models.OneToOneField(DocumentLink)
```

Comment:
@VincentVW  I have updated the docstore Endpoint to '/api/documents/' to avoid any confusion between documentLinks and documents.
Also, I have optimized the uri  '/api/documents/' time execution. it passed from 760 ms to 39 ms in my local machine. I have deleted sql join table between documentLink and document by deleting DocumenLink from Document Serializer. Plz review it. I'll deploy it to dev-post server